src/navigation/EllipsisMenu/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { ClickAwayListener, Divider } from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import React from 'react';
+import styled from 'styled-components';
+
+import { FixedIcon } from '../..';
+
+const MenuWrapper = styled.div`
+  display: flex;
+`;
+
+const MenuItemWrapper = styled.div`
+  :focus {
+    outline: none;
+  }
+`;
+
+const IconWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    margin: 0 4px;
+    border-radius: 50%;
+    transition: background-color .2s ease-in-out;
+    outline: none;
+    height: 24px;
+    width: 24px;
+
+    span {
+      display: flex
+    }
+
+    :hover {
+      background-color: ${({ theme }) => theme.colors.inputField}
+    }
+  }
+`;
+
+export type EllipsisMenuItem = {
+  label: string;
+  disabled?: boolean;
+  onClick: () => void;
+};
+
+type Props = {
+  menuItems: EllipsisMenuItem[];
+};
+
+const EllipsisMenu = ({ menuItems }: Props): React.ReactElement => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = (event: any) => setAnchorEl(event.currentTarget);
```

Comment:
I could not add the type, perhaps someone could help me with it.

Reply:
something around...

```typescript
  const handleClick: React.MouseEventHandler<'onclick'> = (event) => setAnchor(event.currentTarget);
```

seems to work.

Probably too specific, but in this case, I think it's enough.

Reply:
does not work for me

Reply:
![Screenshot from 2020-07-29 14 59 24](https://user-images.githubusercontent.com/16622558/88792363-2abf9480-d1ac-11ea-9f37-b9712e058326.png)

Material UI docs have both JS/TS examples with types :wink:

Reply:
I know and I already test it. But didn't work.

![image](https://user-images.githubusercontent.com/4376509/88806494-e1a91980-d186-11ea-9479-9bc5ea8fed1d.png)

 Could you try yourself? Perhaps it's a problem with my computer.

Reply:
If you read through error message it says that it cannot assign to type of null (which is the inferred type of anchorEl). I already pushed a fix